src/main/components/Browse/Browse.js

```diff
@@ -26,11 +30,122 @@ class Browse extends Component {
 
   constructor(props) {
     super(props);
+
+    this.clearFilters = this.clearFilters.bind(this);
+    this.handleAnalysisClick = this.handleAnalysisClick.bind(this);
+    this.handleFilterChange = this.handleFilterChange.bind(this);
+    this.handleOptions = this.handleOptions.bind(this);
     this.state = {
+      analyses: null,
```

Comment:
I'm still not convinced that analyses should be part of state, because it's calculated by the full set of analyses (from props) and current filters (state).

I would probably opt to take it out of state, and then in render, calculate the filtered analysis list and pass those to the children